docs/config/index.md

```diff
@@ -711,9 +711,14 @@ export default defineConfig({
   Imported or referenced assets that are smaller than this threshold will be inlined as base64 URLs to avoid extra http requests. Set to `0` to disable inlining altogether.
 
   ::: tip Note
-  If you specify `build.lib`, `build.assetsInlineLimit` will be ignored and assets will always be inlined, regardless of file size.
+  If you specify `build.lib` or `build.assetsInlineExclude` , `build.assetsInlineLimit`will be ignored and assets will always be inlined, regardless of file size.
   :::
 
+### build.assetsInlineExclude
+
+- **Type:** `string | RegExp | (string | RegExp)[]`
+- **Default:** `[]`
+
```

Comment:
I think it can add a desc as same as mulit comments, `Exclude paths that are less than the limit but do not want to be included`.